recipes-devtools/python/python3-pycuda_2020.1.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Python wrapper for Nvidia CUDA"
+HOMEPAGE = "http://mathema.tician.de/software/pycuda"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5f3f85642a528c8fca0fe71e7a031203"
+
+SRC_URI = "https://files.pythonhosted.org/packages/46/61/47d3235a4c13eec5a5f03594ddb268f4858734e02980afbcd806e6242fa5/pycuda-${PV}.tar.gz"
+
+SRC_URI[sha256sum] = "effa3b99b55af67f3afba9b0d1b64b4a0add4dd6a33bdd6786df1aa4cc8761a5"
+
+S = "${WORKDIR}/pycuda-${PV}"
+
+inherit  cuda distutils3
+
+DEPENDS = "python3-setuptools-native python3-cython-native python3-cython python3-numpy-native"
+
+RDEPENDS_${PN} += "python3-appdirs python3-decorator python3-mako python3-pytools python3-core \
```

Comment:
You may have build issue because `python3-pytools` is not provided by `poky` or `meta-openembedded`

Reply:
I think we should write the Yocto recipe `python3-pytools`

Reply:
Hmm, yeah that's a problem...  OK, I'll drop this recipe from this PR, and ask Marek to submit it directly.